Codec/Archive/Tar/Write.hs

```diff
@@ -39,6 +43,16 @@ import qualified "os-string" System.OsString.Posix as PS
 write :: [Entry] -> LBS.ByteString
 write es = LBS.concat $ map putEntry es ++ [LBS.replicate (512*2) 0]
 
+write' :: [GenEntry OsPath TarPath LinkTarget] -> IO LBS.ByteString
```

Comment:
I wonder if this function should rather take a `Handle` and just write it, instead of returning `LBS.ByteString`.

Reply:
`cabal-install` does `GZip.compress` before writing